_includes/xlsform-reference-table.md

```diff
@@ -75,8 +75,8 @@ This sheet is mandatory in all XLSForms.
 |                                    	| minimal                              	| Spinner widget - click button to provide a response                                                                               	| yes              	| yes                  	|                 	|                                                                                                                                        	|
 |                                    	| label                                	| Table of answers                                                                                                                  	| yes              	| yes                  	|                 	|                                                                                                                                        	|
 |                                    	| list-nolabel                         	| Goes with label, no label on answer just radios                                                                                   	| yes              	| yes                  	|                 	|                                                                                                                                        	|
-|                                    	| autocomplete                         	| Autocomplete widget                                                                                                               	| no               	| yes                  	|                 	| Does this work?                                                                                                                        	|
-|                                    	| autocomplete_chars                   	| Autocomplete widget with chars option                                                                                             	| no               	| yes                  	|                 	| Does this work?                                                                                                                        	|
+|                                    	| autocomplete                         	| Autocomplete widget                                                                                                               	| no               	| no                  	|                 	
```

Comment:
I think this should be a `yes` for Collect. I feel like I also used it in Enketo recently?